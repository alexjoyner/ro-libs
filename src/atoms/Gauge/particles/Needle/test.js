import React from 'react';
import { shallow } from 'enzyme';
import { Needle } from '.';

describe('Needle component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<Needle />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
