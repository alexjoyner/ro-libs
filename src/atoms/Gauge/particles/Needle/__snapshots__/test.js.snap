// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Needle component Should match snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
    stroke: #34495e;
    stroke-width: 0.3;
    fill: #2980b9;
    transform: rotate(60, 125, 150);
",
        ],
      },
      "displayName": "styled.polygon",
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "polygon",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
